academy/xl-run-prod/6-migration.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Prepare and do migrations
+order: 6
+description: Keep your software and network up to date
+tag: deep-dive
+---
+
+# Prepare and do migrations
+
+Your software is now running, that is good. Time passes and a software upgrade vote proposal passed. You need to upgrade your node to the next version. The state may need to migrate too. When the state needs to migrate, it is also said that there are _breaking changes_. This state migration is the upgrade step that takes most of the time.
+
+You can do the whole process somewhat manually or use a tool to assist you do it smoothly and fast. This is not an unreasonable concern since, when you upgrade, all nodes are simultaneously down.
+
+The main tool is [Cosmovisor](https://docs.cosmos.network/master/run-node/cosmovisor.html). It is a wrapper executable that in turn launches your node as a subprocess and can be configured to stop at a certain point, of time or of block height.
```

Comment:
Can we change all the `https://docs.cosmos.network/master/` link to `https://docs.cosmos.network/main/`

Reply:
I will do it here, but this is something to keep in mind for other pages. @coldice 

```suggestion
The main tool is [Cosmovisor](https://docs.cosmos.network/main/run-node/cosmovisor.html). It is a wrapper executable that in turn launches your node as a subprocess and can be configured to stop at a certain point, of time or of block height.
```